src/Package/Impl/Microsoft.VisualStudio.R.Package.csproj

```diff
@@ -1593,20 +1594,15 @@
       <VSIXSourceItem Include="$(OutputPath)\Microsoft.CodeAnalysis.CSharp.dll" />
       <VSIXSourceItem Include="$(OutputPath)\Microsoft.CodeAnalysis.dll" />
       <VSIXSourceItem Include="$(OutputPath)\libuv.dll" />
-      <VSIXSourceItem Include="$(OutputPath)\zip.dll" />
+      <VSIXSourceItem Include="$(OutputPath)\libzip-5.dll" />
+      <VSIXSourceItem Include="$(OutputPath)\libwinpthread-1.dll" />
```

Comment:
You'll need to add them to low impact setup in RTVS-pr. `Create-FilesCoreSwr.ps1`